### PR TITLE
Add support for message sending.

### PR DESCRIPTION
**Pull request by Czapla-dev**

### CODE REVIEW

**Comment by Czapla-dev** [comment]
Dodano opcje wysyłania wiadomości do kilku nauczycieli.

File: vulcan/_vulcan.py [hunk 1]
```diff
@@ -166,14 +152,23 @@ def get_attendance(self, date_from=None, date_to=None):
     def get_messages(self, date_from=None, date_to=None):
         """
         Fetches messages from the given date
-
         Args:
             date_from (:class:`datetime.date`): Date, from which to fetch messages, if not provided
                 it's using the semester (period) start date
             date_to (:class:`datetime.date`): Date, to which to fetch messages, if not provided
                 it's using the semester (period) end date
-
```
Comment (kapi2289) [inline vulcan/_vulcan.py:175]: Please don't remove these new lines.

File: vulcan/_vulcan.py [hunk 2]
```diff
@@ -166,14 +152,23 @@ def get_attendance(self, date_from=None, date_to=None):
...
         Yields:
             :class:`vulcan._message.Message`
         """
         return Message.get(self._api, date_from, date_to)
+
+    def send_message(self, teachers_id, title, content):
+
+        """
+        sends message to teaches given by id
```
Comment (kapi2289) [inline vulcan/_vulcan.py:168]: Remove the newline before documentation, and start the sentence with a capital letter. Add newlines like in other methods.

File: vulcan/_message.py [hunk 3]
```diff
@@ -85,3 +88,30 @@ def get(cls, api, date_from=None, date_to=None):
                 message["NadawcaId"]
             )
             yield to_model(cls, message)
+
+    @classmethod
+    def send(cls, api, title, content, teachers_id):
+        teachers = []
+        for teacher in teachers_id:
+            teachers_name = (
+                api.dict.get_teacher_json(int(teacher))["Nazwisko"]
```
Comment (kapi2289) [inline vulcan/_message.py:97]: Repeating 3 times the same code is inefficient. Fetch the teacher object once, and then use the variable.

File: vulcan/_vulcan.py [hunk 4]
```diff
@@ -166,14 +152,23 @@ def get_attendance(self, date_from=None, date_to=None):
...
                 it's using the semester (period) start date
             date_to (:class:`datetime.date`): Date, to which to fetch messages, if not provided
                 it's using the semester (period) end date
-
         Yields:
             :class:`vulcan._message.Message`
         """
         return Message.get(self._api, date_from, date_to)
+
+    def send_message(self, teachers_id, title, content):
```
Comment (kapi2289) [inline vulcan/_vulcan.py:165]: I think that `teachers` would be a better parameter name. And it should accept ids, as well as `Teacher` objects (and maybe teacher names too?)

**Review by kapi2289 — CHANGES_REQUESTED** [review]

File: vulcan/_vulcan.py (hunk 1 above)
Comment (Czapla-dev) [reply vulcan/_vulcan.py:175]: I used black to format that file...

**Review by kapi2289 — APPROVED** [review]

**Pull request merged**